arisia-remote/app/arisia/admin/AdminService.scala

```diff
@@ -35,10 +36,13 @@ class AdminServiceImpl(
 ) extends AdminService with Logging {
 
   def getAdmins(): Future[List[LoginId]] = {
+    val column = Fragment.const("admin")
```

Comment:
Is there a reason you aren't using string interpolation here to create a parameterized query? Or does that not work when the parameter is a file name?

https://tpolecat.github.io/doobie-scalaz-0.4.0/05-Parameterized.html

Reply:
Far as I can tell, it doesn't work with column names -- I tried that, and got a fairly obscure error.  It appears to only work in positions where SQL constructed queries allow variables.  This `Fragment.const()` approach appears to be the official solution:

[https://tpolecat.github.io/doobie/docs/08-Fragments.html#composing-sql-literals](https://tpolecat.github.io/doobie/docs/08-Fragments.html#composing-sql-literals)

Reply:
(Note, BTW, that this link is to the current Doobie documentation -- your link above is pretty old, and sounds like it may be Scalaz-specific.  We're using Cats, which is the other category-theory infrastructure; in practice, Doobie is currently agnostic between them.)